src/ray/util/util.cc

```diff
@@ -403,4 +403,23 @@ std::string FormatFloat(float value, int32_t precision) {
   return ss.str();
 }
 
+absl::flat_hash_map<std::string, std::string> GetFinalNodeLabels(
+    const absl::flat_hash_map<std::string, std::string> &labels,
+    const std::string &node_id,
+    const std::string &node_ip) {
+  std::vector<std::string> default_key = {kLabelKeyNodeID, kLabelKeyNodeIP};
+  absl::flat_hash_map<std::string, std::string> final_labels(labels.begin(),
+                                                             labels.end());
+
+  for (const auto &key : default_key) {
+    if (final_labels.contains(key)) {
+      RAY_LOG(WARNING) << "The label key name " << key
+                       << " is used by ray internal and will be overriden.";
+    }
+  }
+  final_labels[kLabelKeyNodeID] = node_id;
+  final_labels[kLabelKeyNodeIP] = node_ip;
```

Comment:
Let's not add node ip as label, we should encourage people to use node id if needed since only that is guaranteed to be unique.

Reply:
Sure, you can always add it later if needed.